mycli/main.py

```diff
@@ -85,12 +94,10 @@ def __init__(self, sqlexecute=None, prompt=None,
         if defaults_file:
             self.cnf_files = [defaults_file]
 
-        default_config = os.path.join(PACKAGE_ROOT, 'myclirc')
-        write_default_config(default_config, '~/.myclirc')
-
-
         # Load config.
-        c = self.config = load_config('~/.myclirc', default_config)
+        c = self.config = read_config_file(self.default_config_file)
+        read_config_files(self.system_config_files, base_config=c)
+        read_config_file(self.user_config_file, base_config=c)
```

Comment:
Mutating `base_config` on each calls feels wrong. Can we collapse this to a single call? 

`self.config = read_config_files(config_files)` where `config_files = [default_config_file, system_config_files, user_config_file]`. 

I realize that `config_files` assignment is not right since it will result in nested list.